Marlin/Marlin_main.cpp

```diff
@@ -8202,6 +8234,21 @@ void manage_inactivity(bool ignore_stepper_queue/*=false*/) {
     if (killCount >= KILL_DELAY) kill(PSTR(MSG_KILLED));
   #endif
 
+  #if ENABLED(CONTINUE_BUTTON) && CONTINUE_PIN != -1
+
+    // Check if the continue button was pressed and wait just in case it was an accidental
+    // key kill key press
+    // -------------------------------------------------------------------------------
+    static int continueCount = 0;   // make the inactivity button a bit less responsive
+    const int CONTINUE_DELAY = 750;
+    if (!READ(CONTINUE_PIN))
+      continueCount++;
+    else if (continueCount > 0)
+      continueCount--;
+
```

Comment:
Ups. Will change the comment later. (Changing code after the second beer is usually bad)